Src/WitsmlExplorer.Frontend/services/mudLogObjectService.tsx

```diff
@@ -0,0 +1,37 @@
+import MudLogObject, { emptyMudLogObject } from "../models/mudLog";
+import { Server } from "../models/server";
+import { ApiClient } from "./apiClient";
+
+export default class MudLogObjectService {
```

Comment:
Same here with removing "Object" from the name

Reply:
Also the file

Reply:
Fixed in WE-768